specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/compute.json

```diff
@@ -337,6 +337,270 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a proximity placement group.": {
+            "$ref": "./examples/CreateAProximityPlacementGroup.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ProximityPlacementGroups"
+        ],
+        "operationId": "ProximityPlacementGroups_Update",
+        "description": "Update a proximity placement group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "proximityPlacementGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the proximity placement group."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ProximityPlacementGroupUpdate"
+            },
+            "description": "Parameters supplied to the Update Proximity Placement Group operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProximityPlacementGroup"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a proximity placement group.": {
+            "$ref": "./examples/PatchAProximityPlacementGroup.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ProximityPlacementGroups"
+        ],
+        "operationId": "ProximityPlacementGroups_Delete",
+        "description": "Delete a proximity placement group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "proximityPlacementGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the proximity placement group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "x-ms-examples": {
+          "Create a proximity placement group.": {
+            "$ref": "./examples/DeleteAProximityPlacementGroup.json"
+          }
+        }
```

Comment:
same LRO question for DELETE as well, please double check in service side.

Reply:
They are not long running operation, final response is returned in sync path